packages/ses-integration-test/test/sanity.test.js

```diff
@@ -0,0 +1,17 @@
+/* global Compartment */
+import test from "tape";
+
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Do you think this lint error might become cumbersome if we make every (nearly every?) package a CJS/ESM/RESM hybrid like SES?

Reply:
The only reason it's an error is because the test is lint-checked after the repo is cleaned (deleting the "main" export).  It should not affect code outside the monorepo, as then the "ses" module is taken from NPM where all the exports have been built.